defaults/main.yml

```diff
@@ -4,5 +4,7 @@
 # This file also serves as a documentation for such a variables.
 
 # Examples of role input variables:
-template_foo: foo
-template_bar: bar
+postgresql_version: "13"
+postgresql_password: "mysecretpassword"
```

Comment:
Is this password really used when not set? If so, that would be a security risk, as users might be lazy (or forget) to  change it, which would lead to a deployment with a known credentials.

Reply:
True, better to set it to `null` in defaults/main.yml

Reply:
...and let the role fail if it stays set to `null`... unless it is handled by ansible already.

There are some concepts to store secrets safely in ansible, like ansible vault, but I'm not sure whether their use in rhel system roles is somewhat limited. Any recommendations from rhel roles experts?

Reply:
We cannot force uses to use ansible-vault, but that's a recommendation. I do not remember if using variabes stored with ansible-vault hides encrypted values them from Ansible output, but I hope it does. In other roles, we use `no_log: true` in tasks that use passwords to hide them from the output, but we are not consistent with this yet.

Reply:
better to use `no_log: true` to be safe

And yes - we recommend the use of Ansible Vault in many other roles for password/passphrase/shared_key/token - fields which require security.  See https://github.com/linux-system-roles/nbde_client#nbde_client_bindings 
```
encryption_password |   | a valid password or passphrase for opening/unlocking the specified device. Recommend vault encrypting the value. See https://docs.ansible.com/ansible/latest/user_guide/vault.html
-- | -- | --
```
Please add some text like that to the README